code/modules/vore/fluffstuff/custom_clothes_vr.dm

```diff
@@ -1811,3 +1811,41 @@ Departamental Swimsuits, for general use
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
 	w_class = ITEMSIZE_NORMAL
 	slot = ACCESSORY_SLOT_OVER
+
+//General definition for bracer items. No icons.
+/obj/item/clothing/accessory/bracer
+	name = "bracer"
+	desc = "A bracer."
+	icon_state = null
+	item_state = null
+	icon_override = null
+	slot_flags = SLOT_GLOVES | SLOT_TIE
+	w_class = ITEMSIZE_SMALL
+	slot = ACCESSORY_SLOT_ARMBAND
+
+//AegisOA:Xander Bevin
+//WanderingDeviant:S'thasha Tavakdavi
+/obj/item/clothing/accessory/bracer/fluff/xander_sthasha
+	name = "Plasteel Bracer"
```

Comment:
Fluff item apps included non digestible features which will be in digest_act_vr.dm

Reply:
I included it specifically for the Plasteel Bracer just lower down, without moving the procs to digest_act_vr, to match Scree's pompom and generally other indigestible fluff items. Ideally, digest_act_vr only has indigestible 'general' items.